dockers/docker-dhcp-relay/cli-plugin-tests/test_config_dhcp_relay.py

```diff
@@ -0,0 +1,175 @@
+import os
+import sys
+import traceback
+from unittest import mock
+
+from click.testing import CliRunner
+
+from utilities_common.db import Db
+
+import pytest
+
+sys.path.append('../cli/config/plugins/')
+import dhcp_relay
+
+@pytest.fixture()
```

Comment:
this fixture is already defined in the conftest.py.

Reply:
Removed